base/strings/basic.jl

```diff
@@ -660,3 +660,12 @@ function last(str::AbstractString, nchar::Integer)
     end
     str[prevind(str, e, nchar-1):e]
 end
+
+# reverse-order iteration for strings and indices thereof
+using Base.Iterators: Reverse
```

Comment:
I believe this line may be causing the CI error.  Perhaps it would be better to fully qualify `Iterators.Reverse` in the lines below?

Reply:
Whoops, I missed that one.  Thanks for finding it!